src/segmentation.py

```diff
@@ -5,7 +5,9 @@ def segment_image(img: np.ndarray):
     """
     Get the colors from an image using KMeans clustering
     :param img: The image to get the colors from
-    :return: (mask, colors) where mask  
+    :return: (mask, colors) where mask is a 2D array of the same shape as img, and colors is a 3x3 array of the colors
+    mask is 0 for the background, 1 where the shape is, and 2 where the letter is
```

Comment:
Good change, was confused in the original version what the function was returning.